piff/des/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 by Mike Jarvis and the other collaborators on GitHub at
+# https://github.com/rmjarvis/Piff  All rights reserved.
+#
+# Piff is free software: Redistribution and use in source and binary forms
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+
+"""
+Some specialized features that specific to the DES survey and the DECam camera.
+"""
+
+from .decam_wavefront import DECamWavefront
+from .decaminfo import DECamInfo
```

Comment:
I moved these imports to here.  I'd rather have all the DES-specific items stay in the piff.des namespace rather than be imported into the full piff namespace.  This will be cleaner once there are multiple camera modules in piff to keep all the DES-specific things out of the main piff namespace.